README.md

```diff
@@ -48,30 +48,31 @@ Because you are viewing a protected topic, you must provide `api_key` and `api_u
 Here is an example of an implementation:
 
 ```python
-engage_path = "/engage"
-engage_docs = EngagePages(
-    parser=EngageParser(
-        api=DiscourseAPI(
-            base_url="https://discourse.ubuntu.com/",
-            session=session,
-            api_key="secretkey", # API KEY used in the tests
-            api_username="canonical",
-        ),
-        index_topic_id=17229,
-        url_prefix=engage_path,
+engage_pages = EngagePages(
+    api=DiscourseAPI(
+        base_url="https://discourse.ubuntu.com/",
+        session=session,
+        get_topics_query_id=14,
+        api_key=DISCOURSE_API_KEY, # replace with your API key
+        api_username=DISCOURSE_API_USERNAME, # replace with correspoding username
     ),
-    document_template="/engage/base.html",
-    url_prefix=engage_path,
-    blueprint_name="engage-pages",
+    category_id=51,
+    page_type="engage-pages", # one of ["engage-pages", "takeovers"]
+    url_prefix="/engage",
+    skip_posts=[] # this is a list of topic ids that we want to exclude from Markdown error checks
```

Comment:
Could we call this `exclude_topics`? "exclude" feels to me like the more standard technical term here. "topics" because we shouldn't confuse what we're calling these things. They're topics right? If it's a topic_id, then it should say "topic" in the variable name.

Reply:
sure, no problem. I was also struggling to find the right word.